src/lib/components/schedule/schedule-form-view.svelte

```diff
@@ -1,13 +1,13 @@
 <script lang="ts">
```

Comment:
Unrelated to this PR, but wondering if we want to make this a bit bigger so `Delete Schedule` fits on one line
![Screenshot 2023-08-29 at 3 59 55 PM](https://github.com/temporalio/ui/assets/15069288/b438ec6a-b2d6-48e3-8da2-15cb28012607)

Reply:
I'll do that on https://github.com/temporalio/ui/pull/1598

Reply:
I think @Alex-Tideman is on this 💪